README.md

```diff
@@ -47,6 +47,37 @@ After that, you may run the binary as any normal program:
 ```
 ./shark-clipper-linux-x64-latest.bin
 ```
+#### Using the Mac Prebuilt Binary
+You will need to set execute permissions on the Mac prebuilt binary, e.g.:
+```
+chmod +x shark-clipper-macos-x64-latest.bin
+```
+
+Mac requires an additional security step to run the prebuilt binary.
+This step needs to be done every time a new binary is downloaded and run for the first time.
```

Comment:
This should be the opening to this section.
This statement also applies to changing the binary permissions.